.github/workflows/solr-almaren-githubactions.yml

```diff
@@ -0,0 +1,45 @@
+name: Solr-Almaren
+on: [push, pull_request]
+
+jobs:
+  Build:
+    runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgres:13.4
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - name : Check out repository code
+        uses: actions/checkout@v2
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 8
+          cache: sbt
+      - name: Setup Solr in Build Machine
+        run: |
+          curl -sSL https://raw.githubusercontent.com/moliware/travis-solr/master/travis-solr.sh | SOLR_VERSION=7.7.2 bash
+          ./solr-7.7.2/bin/solr delete -c gettingstarted
+          ./solr-7.7.2/bin/solr create -c gettingstarted
```

Comment:
Include some sleep 20 secs after collection create step and see pipeline is working or not